subsys/mgmt/mcumgr/lib/cmd/img_mgmt/src/zephyr_img_mgmt.c

```diff
@@ -456,6 +457,12 @@ img_mgmt_impl_erase_image_data(unsigned int off, unsigned int num_bytes)
 
 	LOG_INF("Erased 0x%zx bytes of image slot", erase_size);
 
+#ifdef CONFIG_MCUBOOT_IMG_MANAGER
```

Comment:
This is a trail to mark some MCUboot's strictly related code. I think we might want to decouple image management from  MCUboot at some point - such helpers will make our works easier then.